pkg/router/f5/f5.go

```diff
@@ -828,11 +911,77 @@ func (f5 *f5LTM) Initialize() error {
 		}
 	}
 
+	if f5.setupOSDNVxLAN {
+		err = f5.ensureVxLANTunnel()
+		if err != nil {
+			return err
+		}
+	}
+
 	glog.V(4).Infof("F5 initialization is complete.")
 
 	return nil
 }
 
+func checkIPAndGetMac(ipStr string) (string, error) {
+	ip := net.ParseIP(ipStr)
+	if ip == nil {
+		errStr := fmt.Sprintf("'%s' is not a valid IP address for a vtep", ipStr)
```

Comment:
This implies that the vtep has other restrictions.  Perhaps:
  'vtep IP '%s' is not a valid IP address'